test/libsolidity/syntaxTests/types/mapping/contract_storage_parameter_with_mapping.sol

```diff
@@ -0,0 +1,6 @@
+struct S { mapping(uint => uint)[2] a; }
+contract L {
```

Comment:
Not that it matters that much but when I see `L` in tests I usually expect it to be a library.
```suggestion
contract C {
```

Reply:
Ah, copy-pasting. It matters ). Good catch, thanks!